src/types.ts

```diff
@@ -47,6 +47,10 @@ export interface NotaryToolNotarizeAppOptions {
   appPath: string;
 }
 
+export interface NotaryToolOptions {
+  notarytoolPath?: string;
+}
+
```

Comment:
nit: I think I'd prefer just merging this with the type that accepts `appPath` to avoid having too many single-use interfaces.